### PR TITLE
Polish cassandra setup

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Comment by ktoso** [comment]
Finished prepping cassandra

**Comment by ktoso** [comment]
(feel free to review, just want to get the script on the nodes quickly)

**Comment by bantonsson** [comment]
🇵🇱

File: infra/benchmark-run-sharding-init-cassandra.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+declare -r CLAZZ="com.lightbend.akka.bench.sharding.InitCassandraApp"
+
+fix --verbose node:akka-cassandra-001 ssh:"cd /home/akka/apps; sudo -u akka /home/akka/sbt ';project sharding; runMain $CLAZZ'"
```
Comment (ktoso) [inline infra/benchmark-run-sharding-init-cassandra.sh]: so one would run this one once :)

**Comment by ktoso** [comment]
🇵🇱  cassandra:

```
~$ cqlsh 10.132.0.5
Połączono z akka-cassandra na węźle 10.132.0.5:9042.
```

😉

**Pull request merged**